src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -112,42 +112,42 @@ export class BaseEditCourseForm extends React.Component {
               name="title"
               component={RenderInputTextField}
               type="text"
-              label={<FieldLabel text="Title" required />}
+              label={<FieldLabel text="Title" required requiredForSubmit />}
               required
               disabled={courseInReview}
             />
             <div>
-              <FieldLabel id="number" text="Number" className="mb-2" />
+              <FieldLabel id="number" text="Number" className="mb-2" requiredForSubmit />
```

Comment:
Seems weird to me to have `requiredForSubmit` on a field you can't edit... but old publisher does it so why not

Reply:
I actually hate that old publisher does it. I vote dropping it.